vivisect/codegraph.py

```diff
@@ -148,7 +150,6 @@ def getCodeBlockNode(self, va):
         vaend = valist[vaidx:]
 
         lastva = vabeg[-1]
-        newlastva = vaend[-1]
```

Comment:
this may be cleanup, but based on the bugs i've seen, this may be necessary and simply incomplete.  i'll check it out and do some testing.